### ISSUE TITLE
When will jdk8u282 be released?

### ISSUE THREAD

**Issue by ultimatesecpro** [ojdkbuild/ojdkbuild#127]
When will jdk8u282 be released?
I can find it here https://github.com/ojdkbuild/ojdkbuild/releases/tag/java-1.8.0-openjdk-1.8.0.282-1.b08
But it does not appear here https://github.com/ojdkbuild/contrib_jdk8u-ci/releases

**Comment by ojdkbuild** [comment]
Sorry for the delay, with Travis going away and CentOS 6 repos getting disabled - reconfiguration is required for Linux builds, they are expected to be available before Monday.

**Comment by ojdkbuild** [comment]
CentOS 6 image was easy to solve, new CI config - not so easy. Currently getting interesting problems with Circle CI, will follow up on this.

**Comment by ultimatesecpro** [comment]
May be possible to migrate to CentOS 7?
CentOS 6 End Of Life from November 2020

**Comment by ojdkbuild** [comment]
It is a good question, but I think it will be better to keep the 8u build env the same as before, where possible, and use CentOS 7 for newer jdks (17, not sure about 11).

Cirrus CI setup succeeded - https://github.com/ojdkbuild/contrib_jdk8u-ci/releases/tag/jdk8u282-b08

I will add minor fixes (hashsum missed, cacerts need to be changed - one there is still valid but won't be updated automatically anymore) and will move other jdk-linux builds to Cirrus in a day or two.

**Comment by ultimatesecpro** [comment]
Is it possible to use https://github.com/ojdkbuild/contrib_jdk8u-ci/releases/tag/jdk8u282-b08? 
Or minor fixes is on the way?

**Comment by ojdkbuild** [comment]
Yes, this 8u282 build is fine, minor fixes will be added for the April 8u292 build.
